wiki/Rules/zh.md

```diff
@@ -1,59 +1,56 @@
----
-outdated: true
----
-
 # 规章制度
 
-这些规章制度确保了 osu!社区 能有一个有趣而友好的氛围。
+这些规章制度确保了 osu! 社区能对所有人都有一个有趣而友好的氛围。
 
 ## 社区规定
 
-1. **一个玩家同时只能拥有*一个*帐号。** 这个帐号属于你。请勿将你的帐号与他人共享。
-2. **公平竞争。** 不允许使用第三方工具破坏游戏公平性。包括宏脚本，辅助瞄准程序，调速工具等一切有利于你在游戏中操作的工具。
-3. **待人和善。** 骚扰或敌视他人是不受欢迎的行为，请不要互相伤害。
-4. **不要逃避。** 如果你不确定自己是否违反了规定，可能你已经违反了。
+1. **一个玩家全程只能拥有*一个*帐号。** 你注册的*第一个*帐号是你的唯一的 osu! 账号，从注册开始到永远。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。如果你无法使用该账号，你也不应再创建。保持其安全。
+2. **公平竞争。** 不允许使用任何第三方工具获取任何优势。包括宏脚本，辅助瞄准程序，调速工具等一切有利于你在游戏中操作的工具。任何替你做出游戏内本该你自己做出的动作的程序都不行。
+3. **待人和善。** 骚扰或敌视他人是不受欢迎的行为，请不要互相伤害。我们在这里点圈圈，敲鼓，接水果，或者同时敲最多八个键，而不是互相恶心。
+4. **不要逃避。** 如果你不确定自己是否违反了规定，恐怕你已经违反了。
 5. **我们是全年龄社区。** 这意味着不能发布 R18/NSFW（Not Safe For Work，不宜在工作期间访问）内容，例如毒品，性。除非它们是为了配合语境，否则绝对不能使用。
-6. **规定中没有指出的，请根据常识做出判断。** 管理员有权利给出他们认为合适的裁决。
+6. **规定中没有指出的，请根据常识做出判断。** 管理员有权给出他们认为合适的裁决。
 
 ## 游戏聊天频道规定
 
 1. **尊重聊天频道内的所有玩家，不要刷屏。** 如果你的聊天占据了半个屏幕，别人就不能愉快的聊天了。
-2. **让频道管理员处理聊天内的问题。** 使用`!report`命令，他们收到后会进行处理。
+2. **让频道管理员处理聊天内的问题。** 使用[`!report`命令](/wiki/Reporting_Bad_Behaviour)，他们收到后会进行处理。
 3. **不要讨论被禁言或限制的玩家。** 他的问题只能由他自己和管理员解决，而不是其他人。
-4. **鼓励公平竞争，不要作弊。** 如果你发现了作弊者，请使用`!report`命令举报，或是在他的个人资料页点击举报。
-5. **语言得体，避免 R18/NSFW 内容。** 公共聊天频道不能发布这些内容，但是允许在私聊中发。
+4. **鼓励公平竞争，不要关注作弊玩家。** 很遗憾，作弊玩家确实存在，给予他们过多关注会鼓励他们继续或吸引他人模仿。请使用`!report`命令或是在个人资料页点击来举报他们。
+5. **语言得体，适合全年龄，避免 R18/NSFW 内容。** 公共聊天频道不能发布这些内容，但是允许在私聊中发。
 6. **注意你与他人分享的内容，尤其是可能导致不适的内容。** 分享之前三思而后行。
-7. **确保链接符合法律规定。** 盗版链接，以及“[warez](http://baike.baidu.com/item/WareZ)”相关的内容都不能出现在聊天频道。
-8. **请勿发布第三方聊天工具的链接，包括 Discord，Skype，QQ。** 如果你需要邀请某人加入你的聊天服务器，请在私聊中发。
-9. **在指定了语言的聊天频道，必须使用该语言。** 在没有指定语言的聊天频道，一律使用英文。但是`#lobby`和`#help`允许使用任何语言。*频道管理员在进行工作时可以忽略这条规定。*
+7. **保持链接中的网站相关且合法。** 盗版视频（不包括具有官方授权的视频网站），盗版软件，以及“[warez](http://baike.baidu.com/item/WareZ)”相关的内容都不能出现在聊天频道。可以给游戏直播打广告，但只能在 `#videogames` 频道。
+8. **请勿发布第三方聊天工具的链接，包括 Discord，Skype，QQ。** osu! 聊天频道是关于 osu! 本身的，而不是你的聊天服务器。如果你需要邀请某人，请在私聊中发。
```

Comment:
Just noting here, QQ is listed here when it is not in the English version. Should be fine to leave it as it is.